SR2022/2022-02-10-srawn-07.md

```diff
@@ -0,0 +1,35 @@
+The Student Robotics (Almost) Weekly Newsletter is a (roughly) fortnightly newsletter which summarises the goings‐on across Student Robotics. **Emboldened** items are ones for which additional help has been requested.
+
+## Upcoming Tech Day
+
+After a sucessful [Virtual Tech Day](https://studentrobotics.org/events/sr2022/virtual-tech-day-january/) on the 22nd January in our Discord server. Teams met in person to work on their robots and got help and guidance remotely from our volunteers. Great progess was made!
+
+Following on from this success, we are planning another virtual Tech Day on the 5th March. **If you're interested in helping out with this tech day, please [join #competition-team](https://studentrobotics.slack.com/archives/CBP7UL6RG) in Slack and let us know!**
+
+## Competition Dates Announced
+
+[The competition](https://studentrobotics.org/events/sr2022/competition/) will be held on the 23rd and 24th of April 2022 at SUSU on the University of Southamptons' Highfield Campus. There will be setup days on the 21st and 22nd of April. **If you are interested in volunteering at the competition, please [register your interest](https://docs.google.com/forms/d/e/1FAIpQLSeferj0EDVmPDsaKK-CzG2-YzFJETTbgSeyEMRgdahJ0T3slQ/viewform)** and we'll get in touch closer to the competition to work out the remaining details.
+
+## Tasks have been imported into the task tracker
+
+The recurring tasks that we need to complete for each competition have been imported into [srobo/tasks](https://github.com/srobo/tasks). This year we have grouped the tasks into areas and assigned a Head and a Deputy for each. **If you see a task you want to get involved with just comment on the issue :)**
```

Comment:
Just a thought -- should the link here be to the competition milestone (or something else that's specific to this year, rather than generic)?

Reply:
👍 Maybe `https://github.com/srobo/tasks/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22SR2022+Competition%22`?

Reply:
A shame you can't search for multiple milestones

Reply:
That looks good.
While I'm thinking about this -- do we want to mention that areas have owners/who to get in touch with for each area?

Reply:
[03a7312](https://github.com/srobo/srawn/pull/33/commits/03a731260e352d6a95a71e4bcd922087cae47d4e)